src/dmclock_server.h

```diff
@@ -696,6 +715,7 @@ namespace crimson {
       bool             allow_limit_break;
 
       std::atomic_bool finishing;
+      bool             dynamic_cli_info_f = U1;
```

Comment:
We can remove this variable and still get the use of a well-named variable by using a `constexpr` in your `get_cli_info function` below.